.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+[Back](./README.md)
+
+# Contributing
+
+## A good abbreviation is:
+
+-  `global` • understood by everyone
+-  `significant` • seen for the first time you still know what it means
+
+## Open an issue
+- [New abbr](./ISSUE_TEMPLATE/newAbbr.MD)
+- [Edit curr abbr](./ISSUE_TEMPLATE/editCurrAbbr.md)
```

Comment:
Shall we really need to abbreviate here? :smile: 

```suggestion
- [New abbreviation](./ISSUE_TEMPLATE/newAbbr.MD)
- [Edit current abbreviation](./ISSUE_TEMPLATE/editCurrAbbr.md)
```

Reply:
thought it was funny to use the abbr we state for codes

Reply:
Yeah, I see you point and it's really funny :slightly_smiling_face: 

Ok, let's keep your suggestion!

Reply:
HAHA ok👌